lib/rust/ast/impl/build.rs

```diff
@@ -0,0 +1,25 @@
+use std::fs::File;
+use std::io::prelude::*;
+use ast_generation::api;
+
+
+/// Generates the AST API  and saves the result into the file `src/api.rs`.
+///
+/// The content of the generated file can be used with the `include!` macro.
```

Comment:
It shouldn't be used with the `include!` macro. Just generate the file in place instead, like the lexer does.